explorer/explorer.go

```diff
@@ -1563,6 +1565,16 @@ func (exp *Service) GetStateRootHash(blockHeight int64) (string, error) {
 	return hex.EncodeToString(rootHash[:]), nil
 }
 
+// GetMeta gets meta data of a blockchain
+func (exp *Service) GetMeta() (explorer.Meta, error) {
+	var meta explorer.Meta
+	meta.ChainID = int64(exp.bc.ChainID())
+	meta.ProtocolVersion = version.ProtocolVersion
```

Comment:
could we keep the two version consistent and put it into GetServerMeta if we do want to add it to v1?